tests/unit/test_forms.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from django.test import override_settings
+
+from wagtailtrans.models import Language
+from wagtailtrans.forms import LanguageForm
+
+@pytest.mark.django_db
+class TestLanguageForms(object):
+
+    @override_settings(WAGTAILTRANS_SYNC_TREE=True)
+    def test_post_first_language(self):
+        Language.objects.all().delete()
+        data = {
+            'code': 'en',
+            'is_default': True,
+            'position': 0,
+            'live': True
+        }
+        form = LanguageForm(data)
+        assert form.is_valid()
```

Comment:
Let's check if the language is actually set as default language here. `instance = form.save()`.
Now we're only validating if the form is valid, not what's going to be saved.